src/managers/UserManager.ts

```diff
@@ -5,7 +5,7 @@ import {MongoManager} from "./MongoManager";
 import {IIdNameInfo, IUserInfo} from "../definitions";
```

Comment:
This caused problems when I tried to replace my nicknames.  Consider handling things like "null" for rawName, or consider why null is being passed into this function in the first place.  When I checked for null and returned "", getAllNames crashed because of null.  Just review these methods and how they are being called.